fecgov/fec-cms
# Feature/4698 fix purgecss

## Summary

- Resolves #4698 

After adding purgecss, we lost styles for the interactive parts of the homepage—the menu wouldn't open and search suggestions were jumbled

### Required reviewers

2

## Impacted areas of the application

The homepage, mostly the interactive components

## Screenshots

The before shots are in the ticket #4698 . The 'after' should look like Production

## Related PRs

Nah

## How to test

- pull the branch
- `npm i` just in case
- `npm run build-sass`
- check [localhost homepage](http://127.0.0.1:8000) to make sure it looks like Prod
- check the interactive parts, too
   - main nav - wide
   - main nav - small
   - search results (make sure you get committee suggestions to see the active/inactive dots)
   - feedback
   - glossary
   - maps
- `npm run build`
   - check the same components
- let me know if things are ugly or invisible
